plugins/tide-display/manager.py

```
@@ -66,7 +66,7 @@ def _lerp(c1, c2, t):
 
 def _safe_iso(s):
     try:    return datetime.fromisoformat(s)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)